docs/resources/Guild.md

```diff
@@ -230,9 +233,14 @@ Create a new [channel](#DOCS_CHANNEL/guild-channel-object) object for the guild.
 | user_limit | integer | the user limit of the voice channel (voice only) |
 | permission_overwrites | an array of [overwrite](#DOCS_CHANNEL/overwrite-object) objects | the channel's permission overwrites |
 
-## Modify Guild Channel Position % PATCH /guilds/{guild.id#DOCS_GUILD/guild-object}/channels
+## Modify Guild Channel Positions % PATCH /guilds/{guild.id#DOCS_GUILD/guild-object}/channels
+
+Modify the position of channels in a guild. Requires 'MANAGE\_CHANNELS' permission. Returns a 204 empty response on success. Fires a [Channel Update](#DOCS_GATEWAY/channel-update) Gateway event.
+
+>info
+> Only affected channels are required, the minimum being a swap between two channels.
 
-Modify the position of a guild channel. Requires 'MANAGE\_CHANNELS' permission. Returns a 204 empty response on success. Fires a [Channel Update](#DOCS_GATEWAY/channel-update) Gateway event.
+This endpoint takes a JSON array of parameters in the following format:
```

Comment:
There was also in "Batch Modify Guild Role", which is now "Modify Guild Role Positions". Since the root type in this case is an array instead of the usual object, I think this should be left in.